casepro/msgs/forms.py

```diff
@@ -79,3 +80,45 @@ def clean_keywords(self):
     class Meta:
         model = Label
         fields = ('name', 'description', 'is_synced', 'keywords', 'groups', 'field_test', 'ignore_single_words')
+
+
+class FaqForm(forms.ModelForm):
+
+    question = forms.CharField(label=_("Question"), max_length=140)
+    answer = forms.CharField(label=_("Answer"), max_length=140)
+    language = forms.CharField(label=_("Language"), max_length=3)
+    # limit the parent choices to FAQs that have a ForeignKey parent that is None
+    parent = forms.ModelChoiceField(queryset=FAQ.objects.filter(parent=None), required=False)
```

Comment:
This is handled here: https://github.com/praekelt/casepro/blob/EPIC-OPENHELPD-15-PR/casepro/msgs/views.py#L496-L502